src/App.js

```diff
@@ -17,9 +17,31 @@ class App extends Component {
   }
 
   componentDidMount() {
-    menData.forEach(this.addToLeaderboard);
+    fetch(API+'Masculino')
+    .then(res => res.json())
+    .then(data => { data.forEach(this.addToLeaderboard); })
+    .catch(err => { console.log('Error happened during fetching!', err); });
+  }
+
+  ordinal_suffix(i) {
```

Comment:
Remember to follow javascript standards. In this case, use snakeCase

Reply:
I think a better refactor is
```js
const j = i % 10;
const k = i % 100;
let suffix = "th";

if (j === 1 && k !== 11) {
  suffix = "st";
} else if (j === 2 && k !== 12) {
  suffix = "nd";
} else if (j === 3 && k !== 13) {
  suffix = "rd";
}

return `${i}${suffix}`;
```
Rename the variables `i`, `j` and `k` for a better names. I could not identify the purposes of them quickly. Also, with this version of ECMAScript, you use `const` or `let` instead of `var`